components/config-mgmt-service/backend/elastic/nodes.go

```diff
@@ -238,19 +238,26 @@ func (es Backend) GetNodes(page int, perPage int, sortField string,
 }
 
 // GetNodesCounts - get the number of successful, failure, and missing nodes
-func (es Backend) GetNodesCounts(filters map[string][]string) (backend.NodesCounts, error) {
+func (es Backend) GetNodesCounts(filters map[string][]string,
+	startDate string, endDate string) (backend.NodesCounts, error) {
 	var ns = *new(backend.NodesCounts)
 
 	// Adding the exists = true filter to the list of filters, because nodes
 	// have documents that persist in elasticsearch to hold historical data
 	// even after the node no longer exists
 	filters["exists"] = []string{"true"}
 
-	boolQuery := newBoolQueryFromFilters(filters)
+	mainQuery := newBoolQueryFromFilters(filters)
```

Comment:
This is the main change with this PR. The rest is passing the start and end date to this function.